build.xml

```diff
@@ -145,6 +147,20 @@
         <fail if="tests.failed" message="There were failed unit tests"/>
     </target>
 
+    <target name="test" depends="compile, set_excluded_test_groups" description="Run unit tests">
+        <antcall target="run-test">
+            <param name="excludedTestGroups" value="${excludedTestGroups}, sra"/>
```

Comment:
What if `excludedTestGroups` is empty? Then you'd get `value=", sra"`. Will this be ok?